R/otp-plan.R

```diff
@@ -424,12 +424,12 @@ otp_parse_missing <- function(x){
 
 otp_clean_input <- function(imp, imp_name) {
   # For single point inputs
-  if (all(class(imp) == "numeric")) {
+  if (!is.matrix(imp) && is.double(imp)) {
```

Comment:
this test looked to me like "non-matrix numeric vector", which matches the new test. CMIIW.